ckanext/scheming/templates/scheming/display_snippets/datetime.html

```diff
@@ -0,0 +1,2 @@
+{% set url_parts = data[field.field_name].split() %}
+{{ url_parts[0] }} {{ url_parts[1] }}
```

Comment:
How about replacing it with `{% extends 'scheming/display_snippets/text.html' %}` which would allow us to overwrite it if we need?